sample-functions/SentimentAnalysis/handler.py

```diff
@@ -1,17 +1,31 @@
 import sys
+import json
 from textblob import TextBlob
 
+reload(sys)
+sys.setdefaultencoding('utf8')
+
 def get_stdin():
     buf = ""
     for line in sys.stdin:
         buf = buf + line
-    return buf
+        return buf
 
 if(__name__ == "__main__"):
     st = get_stdin()
     blob = TextBlob(st)
-    out =""
+    res = {
+        "polarity": 0,
+        "subjectivity": 0
+    }
+
     for sentence in blob.sentences:
-        out = out + "Polarity: " + str(sentence.sentiment.polarity) + " Subjectivity: " + str(sentence.sentiment.subjectivity)  + "\n"
-    print(out)
+        res["subjectivity"] = res["subjectivity"] + sentence.sentiment.subjectivity
```

Comment:
This (and the one below) can be changed to this to follow DRY

```python
res["subjectivity"] += sentence.sentiment.subjectivity
res["polarity"] = += sentence.sentiment.polarity
```

You can also use `/=` for division:

```python
res["polarity"] /= total
res["subjectivity"] /= total
```